builtin/providers/aws/resource_aws_network_interface_test.go

```diff
@@ -108,6 +128,10 @@ func testAccCheckAWSENIAttributes(conf *ec2.NetworkInterface) resource.TestCheck
 			return fmt.Errorf("expected private ip to be 172.16.10.100, but was %s", *conf.PrivateIPAddress)
 		}
 
+		if *conf.SourceDestCheck != true {
```

Comment:
The `TestAccAWSENI_sourceDestCheck` test added does not call this method. Without the mentioned `Default: true,`, the `TestAccAWSENI_basic` test fails. Throw that default in there and I think we're all good :smile: